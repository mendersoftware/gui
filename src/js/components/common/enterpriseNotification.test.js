import React from 'react';
import renderer from 'react-test-renderer';
import EnterpriseNotification from './enterpriseNotification';

describe('EnterpriseNotification Component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<EnterpriseNotification benefit="have a test benefit" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
