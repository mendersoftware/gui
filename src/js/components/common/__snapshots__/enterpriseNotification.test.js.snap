// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EnterpriseNotification Component renders correctly 1`] = `
<p
  className="info icon"
>
  <svg
    aria-hidden="true"
    className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
    focusable="false"
    role="presentation"
    style={
      Object {
        "margin": "0 6px 4px 0",
        "verticalAlign": "middle",
      }
    }
    viewBox="0 0 24 24"
  >
    <path
      d="M11 7h2v2h-2zm0 4h2v6h-2zm1-9C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z"
    />
  </svg>
  With a commercial Mender plan you can have a test benefit. 
  <a
    href="https://mender.io"
    target="_blank"
  >
    Learn more
  </a>
</p>
`;
